utils/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
```

Comment:
If we do decide to have this extra crate, shouldn't it be a child of the `crates` directory?